source/reference/method/MongoDBClient-bulkWrite.txt

```diff
@@ -0,0 +1,99 @@
+============================
+MongoDB\\Client::bulkWrite()
+============================
+
+.. meta::
+   :description: Perform write operations across multiple namespaces by using the MongoDB PHP Library.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 2.1
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\Client::bulkWrite()
+
+   Perform multiple write operations across multiple namespaces.
+
+   .. code-block:: php
+
+      function bulkWrite(
+          BulkWriteCommand|ClientBulkWrite $bulk,
+          array $options = []
+      ): MongoDB\BulkWriteCommandResult
+
+Parameters
+----------
+
+``$bulk`` : :phpclass:`MongoDB\ClientBulkWrite` or
+  :php:`BulkWriteCommand <mongodb-driver-bulkwritecommand>`
+
+  .. tip:: Prefer ClientBulkWrite API
+
+     We recommend using the ``ClientBulkWrite`` builder class and
+     methods to specify write operations in a bulk write command instead
+     of using the ``BulkWriteCommand`` class.
```

Comment:
It's better with this note.

You can give an argument why we recommend it: the `ClientBulkWrite` provides a fluent API with method similar to the Collection class.